### PR TITLE
Add Github CI

### PR DESCRIPTION
**Pull request by payano**
This is working on my master branch, I just created a new branch to collect all changes and make it into one commit.

I think we need to add the CMake Action under "Actions" in the github tab in order to make the test run on every commit. I don't have permissions on this repository, but maybe you can add it?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-Travis CI [![Travis CI Build Status](https://travis-ci.com/lensfun/lensfun.svg?branch=master)](https://travis-ci.com/lensfun/lensfun)
-
-AppVeyor [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/lensfun/lensfun?branch=master&svg=true)](https://ci.appveyor.com/project/seebk/lensfun)
+Github [![CMake](https://github.com/payano/lensfun/actions/workflows/cmake.yml/badge.svg)](https://github.com/payano/lensfun/actions/workflows/cmake.yml)
```
Comment (payano) [inline README.md:1]: This URL needs to be changed to the correct one.

Comment (payano) [reply README.md]: Done.

**Comment by seebk** [comment]
Awesome, it directly ran the jobs inthe Actions tab after merging the PR :)

One minor issue seems to be the badge in the README is not reporting the status. But maybe we just have to wait a bit more...

**Comment by payano** [comment]
I can see that the badge is working?

**Pull request merged**